contributing/README.md

```diff
@@ -0,0 +1,130 @@
+# How to contribute
+
+The Django Girls Organizer Manual is licensed under a [*Creative Commons Attribution-ShareAlike 4.0*](https://creativecommons.org/licenses/by-sa/4.0/) license. Everyone is free to add, edit and correct the tutorial.
+
+# Editing basics
+
+The source code of the organizer manual is [hosted on GitHub](https://github.com/DjangoGirls/organizer-manual). The GitHub [Fork & Pull workflow](https://help.github.com/articles/using-pull-requests) is used to accept and review changes.
+
+The organizer manual uses the [HonKit](https://github.com/honkit/honkit) project for publishing its documentation. [See more information about how HonKit works](https://honkit.netlify.app/).
+
+The organizer manual is written in [Markdown mark up language](https://help.github.com/articles/markdown-basics).
+
+You can find any discussions about the contents of the organizer manual on the [GitHub issue tracker](https://github.com/DjangoGirls/organizer-manual/issues).
+
+[Crowdin](https://crowdin.com/project/django-girls-organizer-manual) platform is used to manage translations. If you want to join an existing translation team or launch a new translation, send an email to the [translation managers](mailto:translations@djangogirls.org) or contact [support team](mailto:hello@djangogirls.org). If you want to propose some small changes or fix typos in existing translations, please create a Pull Request.
```

Comment:
Is https://crowdin.com/project/django-girls-organizer-manual the correct address? It gives me

> # Something went wrong
>
> Project does not exist or access to this project is limited.
>
> In case you are confident that you have access to the project, try to switch account if you have several or contact a project manager for further assistance.

Reply:
Thanks for that @das-g. I hadn't set it up but it's now set up.